docs/how_tos/i18n.rst

```diff
@@ -184,8 +184,26 @@ Create your pipeline job on Jenkins
 
      * Note that the Jenkins machine uses UTC.
 
+#. Test your change locally.
+
+   * Clone https://github.com/openedx/ecommerce-scripts.
+
+   * Create virtualenv for python.
+
+   * Run ``pip install -r transifex/requirements/microfrontend-common.txt``.
```

Comment:
Not all repos use the microfrontend-common.txt. Some repos, for example, edx-analytics-dashboard, uses its own defined requirement.txt.
You can use this as an example for sure, but please provide some instructions on repos using their own requirements.txt